src/genie/libs/parser/iosxe/show_cts_pacs.py

```diff
@@ -0,0 +1,130 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ==================
+# Schema for:
+#  * 'show cts pacs'
+# ==================
+class ShowCtsPacsSchema(MetaParser):
+    """Schema for show cts pacs."""
+
+    schema = {
```

Comment:
not big issue. but if possible, I want to suggest below schema close to actual output based on indentation.
```
    schema = {
        "aid": str,
        "pac_info": {
            "aid": str,
            "pac_type": str,
            "i_id": str,
            "a_id_info": str,
            "credential_lifetime": str,
        },
        "pac_opaque": str,
        "refresh_timer": str
    }

Reply:
@tahigash, I see your point. I believe this schema will add an unnecessary level to the dictionary. Based on the example output provided, I think this schema is sufficient. It's also personal preference. If you feel like the example schema you provided is better suited, I'd be more than happy to accommodate. Let me know. Thanks!

Reply:
if so, can I ask you to adjust to suggested one?

Reply:
if so, can I ask you to adjust to suggested one?

Reply:
@tahigash, this was updated to the sugguested schema.

Reply:
confirmed!